css/about.css

```diff
@@ -0,0 +1,278 @@
+.about-container {
+  background-color: var(--color-lighter-gray);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about-hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0 12px;
+  padding-top: 10px;
+  padding-bottom: 30px;
+}
+
+.about-hero h4 {
+  font-size: 25px;
+  color: var(--color-royal-orange);
+  font-weight: 500;
+  margin-bottom: 10px;
+}
+
+.about-hero h1 {
+  font-size: 32px;
+  font-weight: 900;
+  letter-spacing: 2px;
+  line-height: 35px;
+  color: var(--color-royal-orange);
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+.about-intro {
+  background-color: var(--color-white);
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 25px;
+  letter-spacing: -6%;
+  text-align: center;
+  padding: 25px 10px;
+  color: var(--color-black);
+}
+
+.about-contact {
+  background-color: var(--color-lighter-gray);
+  padding: 20px 30px;
+}
+
+.about-contact p {
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 25px;
+  text-align: center;
+  color: var(--color-text-gray);
+}
+
+.about-contact h4 {
+  font-weight: 800;
+  font-size: 12px;
+  line-height: 25px;
+  letter-spacing: 2%;
+  text-align: center;
+  color: var(--color-gray);
+  text-decoration: underline;
+  text-underline-offset: 5px;
+}
+
+.logo-info,
+.previous-summit-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+  padding: 30px 10px;
+  background-color: var(--color-white);
+}
+
+.logo-info-heading h2,
+.previous-summit-info h2 {
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 28px;
+  letter-spacing: -2%;
+  text-align: center;
+  color: var(--color-dark-charcoal);
+}
+
+.logo-underline {
+  border-bottom: 2px solid #a04038;
+  width: 40px;
+}
+
+.logo-info p,
+.previous-summit-info p {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -3%;
+  text-align: center;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-top: 1px solid var(--color-light-gray);
+  border-bottom: 1px solid var(--color-light-gray);
+}
```

Comment:
- [ ] Good job so far :clap: , However please ensure that you follow design requirement given, as your Logo conatiner design matches requirement.  Please resolve it. :slightly_smiling_face: 

Below is an illustrations of yours and the given footer Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student about section rectangular](https://github.com/chiedozie013/TechTitans-Global-Summit-2024/assets/43528594/477cb364-2db3-4d06-9a7c-094265afc552)

</td>
      <td>

![Design About Section Rectangular](https://github.com/chiedozie013/TechTitans-Global-Summit-2024/assets/43528594/7647b3c3-a0c1-4e5b-8aa4-620cc20267bc)

</td>
    </tr>
  </table>